template/docs/advanced/partial-dev-env.md

```diff
@@ -0,0 +1,55 @@
+# Partially Set Up Development Environment
+
+In certain cases, it is unnecessary to install all dependencies as well as the pre-commit hook. For example, speed up the setup process in CI/CD.
```

Comment:
```suggestion
In certain cases, it is unnecessary to install all dependencies as well as the pre-commit hook. For example, this can speed up the setup process in CI/CD.
```